backend/submissions/admin.py

```diff
@@ -83,7 +83,14 @@ class SubmissionAdmin(AdminUsersMixin):
         (_("Speaker"), {"fields": ("speaker_level", "previous_talk_video")}),
     )
     list_filter = ("conference", "type", "topic", "status")
-    search_fields = ("title", "abstract")
+    search_fields = (
+        "title",
+        "elevator_pitch",
+        "abstract",
+        "notes",
+        "previous_talk_video",
+        "comments__text",
```

Comment:
do we think it might be helpful to search the comments too?

Reply:
Maybe when you only remember "hey someone has commented to that pr about...."  ? 
I don't know tbh, I added anything that can be helpful but I should ask to the people that will use this admin...

Reply:
well I can remove it and we can add if it's needed...